checker/src/main/java/org/checkerframework/checker/signedness/jdk.astub

```diff
@@ -3313,29 +3313,94 @@ class DigestOutputStream {
+    public static int binarySearch(@PolySigned short[] a, int fromIndex, int toIndex, @PolySigned short key);
+    public static int binarySearch(@PolySigned int[] a, int fromIndex, int toIndex, @PolySigned int key);
+    public static int binarySearch(@PolySigned long[] a, int fromIndex, int toIndex, @PolySigned long key);
+    public static int binarySearch(@PolySigned char[] a, int fromIndex, int toIndex, @PolySigned char key);
+
+    public static @PolySigned byte[] copyOf(@PolySigned byte[] original, int newLength);
+    public static @PolySigned short[] copyOf(@PolySigned short[] original, int newLength);
+    public static @PolySigned int[] copyOf(@PolySigned int[] original, int newLength);
+    public static @PolySigned long[] copyOf(@PolySigned long[] original, int newLength);
+    public static @PolySigned char[] copyOf(@PolySigned char[] original, int newLength);
+
+    public static @PolySigned byte[] copyOfRange(@PolySigned byte[] original, int from, int to);
+    public static @PolySigned short[] copyOfRange(@PolySigned short[] original, int from, int to);
+    public static @PolySigned int[] copyOfRange(@PolySigned int[] original, int from, int to);
+    public static @PolySigned long[] copyOfRange(@PolySigned long[] original, int from, int to);
+    public static @PolySigned char[] copyOfRange(@PolySigned char[] original, int from, int to);
+
+    public static boolean equals(@PolySigned byte[] a, @PolySigned byte[] a2);
+    public static boolean equals(@PolySigned short[] a, @PolySigned short[] a2);
+    public static boolean equals(@PolySigned int[] a, @PolySigned int[] a2);
+    public static boolean equals(@PolySigned long[] a, @PolySigned long[] a2);
+    public static boolean equals(@PolySigned char[] a, @PolySigned char[] a2);
+
     public static void fill(@PolySigned byte[] a, @PolySigned byte val);
-    public static void fill(@PolySigned char[] a, @PolySigned char val);
     public static void fill(@PolySigned short[] a, @PolySigned short val);
     public static void fill(@PolySigned int[] a, @PolySigned int val);
     public static void fill(@PolySigned long[] a, @PolySigned long val);
+    public static void fill(@PolySigned char[] a, @PolySigned char val);
     public static void fill(@PolySigned byte[] a, int fromIndex, int toIndex, @PolySigned byte val);
-    public static void fill(@PolySigned char[] a, int fromIndex, int toIndex, @PolySigned char val);
     public static void fill(@PolySigned short[] a, int fromIndex, int toIndex, @PolySigned short val);
     public static void fill(@PolySigned int[] a, int fromIndex, int toIndex, @PolySigned int val);
     public static void fill(@PolySigned long[] a, int fromIndex, int toIndex, @PolySigned long val);
+    public static void fill(@PolySigned char[] a, int fromIndex, int toIndex, @PolySigned char val);
 
-    public static @PolySigned byte[] copyOfRange(@PolySigned byte[] original, int from, int to);
```

Comment:
The reorderings you did in this file, and also its extremely large size, indicate that we should create a real annotated JDK rather than continuing to expand a stub file.  That will be increasingly true as you add more JDK annotations.

Can you please convert this stub file into an annotated JDK?  You can find instructions at https://checkerframework.org/manual/#annotating-jdk .  Be sure to retain both annotations and comments (that are related to the Signedness Checker) from this stub file.  You could do that after or before merging this pull request.

Thanks!